Firmware/RTK_Everywhere/Begin.ino

```diff
@@ -1282,7 +1301,18 @@ void deleteSDSizeCheckTask()
 }
 
 // =-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-
+// Housekeeping
+void housekeepingSetup()
+{
+    clearRegisteredCorrectionsSources(); // Clear (initialize) the vector of corrections sources. Probably redundant...?
+}
 
+void housekeepingLoop()
```

Comment:
I don't like the name of this function. It does not tell me what it does. I suspect this is updateCorrectionsPriorities() or something akin.